deploy/configs/packages.yaml

```
@@ -59,6 +59,9 @@ packages:
             m4@1.4.16: /usr
         version: [1.4.16]
     # can not be installed with python 3
+    mvapich2:
+        variants: ch3_rank_bits=32 fabrics=mrail file_systems=ime,gpfs,nfs,ufs process_managers=slurm threads=multiple ~alloca~cuda~debug+regcache
```

```suggestion
        variants: fabrics=mrail file_systems=ime,gpfs process_managers=slurm
```
Seems like that should be enough?

---

Sure, I added those options because they were on my `.spack/packages.yaml`.